packages/salesforcedx-vscode-visualforce/package.nls.ja.json

```diff
@@ -0,0 +1,22 @@
+{
+  "configuration.title": "Salesforce Visualforce Configuration",
+  "visualforce.format.enable.desc": "デフォルトのVisualforce フォーマッターの有効化/無効化",
+  "visualforce.format.wrapLineLength.desc": "1行あたりの最大の文字数 (0は折り返し無効化)",
+  "visualforce.format.unformatted.desc": "再フォーマットしないタグをカンマ区切りでリストします。何も指定しない場合デフォルトで、https://www.w3.org/TR/html5/dom.html#phrasing-content に一覧表示されているすべてのタグ対象になります。",
+  "visualforce.format.contentUnformatted.desc": "コンテンツをフォーマットしないタグをカンマ区切りでリストします。何も指定しない場合デフォルトで 'pre' タグが対象となります。",
+  "visualforce.format.indentInnervisualforce.desc": "<head> タグと <body> タグをインデントします。",
+  "visualforce.format.preserveNewLines.desc": "要素の前に空行を許可するかどうか。要素の前のみ有効となり、タグの内部やテキストに対しては無効です。",
+  "visualforce.format.maxPreserveNewLines.desc": "許可する空行の最大数。何も指定しない場合デフォルトで上限はありません。",
```

Comment:
English is "Use 'null' for unlimited." 
Was wondering if you would actually enter 'null' in the field or leave it empty.

Reply:
The text box in this setting allows only numbers or blank. So users cannot input text string.